### ISSUE TITLE
Db handle leak

### ISSUE THREAD

**Issue by dan-compton** [cloudquery/cloudquery#28]
It appears that the the closure of Rows is not deferred nor is the last error returned by the function. please dismiss this of incorrect otherwise I'll open up a PR later to fix this and other resource leaks likes this they can be picked up by static analysis tooling or manually.

**Comment by dan-compton** [comment]
https://github.com/cloudquery/cloudquery/blob/main/cloudqueryclient/client.go#L193

This occurs in the implementation of run query which also potentially panics I think if no rows are returned in the initial query

**Comment by yevgenypats** [comment]
Good catch. will be happy for a PR. I think in other places I use  `gorm` `CREATE` where you don't need to call `close`

**Comment by yevgenypats** [comment]
Fixed.
